app/src/main/java/com/humberto/tasky/agenda/presentation/edit_text/EditTextAction.kt

```diff
@@ -0,0 +1,6 @@
+package com.humberto.tasky.agenda.presentation.edit_text
+
+interface EditTextAction {
+    data object OnBackClick: EditTextAction
+    data class OnSaveClick(val content: String): EditTextAction
```

Comment:
Nit - It pays off in the long run to give the best names to parameters, classes, functions, and variables. I believe it is a genuinely underestimated part of writing code. Descriptive names help future developers onboard and understand better the intent of some code without making guesses or performing manual testing. That future developer could even be you coming back to a codebase that you haven’t worked on for a couple of years 😄 

How about a name like: `newText` or `updatedText`?